### PR TITLE
Migrate dependencies

### PR DESCRIPTION
**Pull request by Thog**
Migrate our dependencies to latest versions.

### CODE REVIEW

File: src/pki.rs [hunk 1]
```diff
@@ -80,15 +82,15 @@ impl EncryptedKeyblob {
...
         cmac.verify(&self.0[..0x10])
             .map_err(|err| (keyblob_id, err))?;
 
-        let mut crypter = Ctr128::<Aes128, ZeroPadding>::new_fixkey(
+        let mut crypter = Ctr128::<Aes128>::new(
             GenericArray::from_slice(&key.0),
             GenericArray::from_slice(&self.0[0x10..0x20]),
         );
-        crypter.decrypt_nopad(&mut keyblob)?;
+        crypter.apply_keystream(&mut keyblob);
```
Comment (Thog) [inline src/pki.rs:93]: Not sure if it's the right thing to do here.

File: src/pki.rs [hunk 2]
```diff
@@ -56,15 +58,15 @@ impl Keyblob {
         let mut encrypted_keyblob = [0; 0xB0];
         encrypted_keyblob[0x20..].copy_from_slice(&self.0);
 
-        let mut crypter = Ctr128::<Aes128, ZeroPadding>::new_fixkey(
+        let mut crypter = Ctr128::<Aes128>::new(
             GenericArray::from_slice(&key.0),
             GenericArray::from_slice(&encrypted_keyblob[0x10..0x20]),
         );
-        crypter.encrypt_nopad(&mut encrypted_keyblob[0x20..])?;
+        crypter.apply_keystream(&mut encrypted_keyblob[0x20..]);
```
Comment (Thog) [inline src/pki.rs:65]: Not sure if it's the right thing to do here.

Comment (roblabla) [reply src/pki.rs:65]: Yeah this is correct. Aes CTR is a stream cipher, there never was any padding to begin with!

File: src/pki.rs (hunk 1 above)
Comment (roblabla) [reply src/pki.rs:93]: Yeah this is correct. Aes CTR is a stream cipher, there never was any padding to begin with!

**Review by roblabla — APPROVED** [review]

**Pull request merged**